styx-client/src/main/java/com/spotify/styx/client/auth/GoogleIdTokenAuth.java

```diff
@@ -70,9 +88,24 @@ private String getToken(String targetAudience, GoogleCredential credential)
     }
   }
 
+  private String getDefaultGCEIdToken(String targetAudience) throws IOException {
```

Comment:
Does this cover other non-materialized keys situations? Maybe the metadata server situation is the only one, not sure.

Reply:
No, but this PR is only about the GCE default credentials (via metadata server) use case.

Reply:
I misunderstood the problem when creating Jira ticket. Supporting non-materialized keys situation is more urgent than support GCE default.

Reply:
Yes, styx client & cli should work with our internal `toogle` without this PR.